web-platform-tests/wpt.fyi
# Remove self-navigator in wpt-runs page. Delete self-navigator.js

Self-Navigator is no longer used. It was replaced by app-route in #1295. I think it was missed when removing it from other files. This removes its usage from the wpt-runs page

Way to test that navigation still works:
- Navigate to "Recent Runs"
- Click any link from that page. It should navigate successfully.

Context: Was trying to debug when the manual "page_view" event was being triggered in self-navigator.js. My breakpoint never triggered. Did research and saw that navigation was handled by app-route instead.
